Update Merge testnet example to longterm networks (#1108)
hyperledger/besu-docs: docs/Tutorials/Merge-Testnet.md

Description: How to run Besu and Teku on the Merge testnet
---

> # Run Besu and Teku on a post-merge testnet

Reviewer: ```suggestion
# Run Besu and Teku on a post-Merge testnet
```

You can test Besu as an [execution client](../Concepts/Merge.md#execution-clients) and
[Teku](https://docs.teku.consensys.net/en/stable/)

> as a [consensus client](../Concepts/Merge.md#consensus-clients) on a post-merge testnet.

Reviewer: ```suggestion
as a [consensus client](../Concepts/Merge.md#consensus-clients) on a post-Merge testnet.
```

    ```

=== "Sepolia"

    ```bash
    besu \
      --network=sepolia           \
      --rpc-http-enabled=true     \
      --rpc-http-host="0.0.0.0"   \
      --rpc-http-cors-origins="*" \
      --rpc-ws-enabled=true       \
      --rpc-ws-host="0.0.0.0"     \
      --host-allowlist="*"        \
      --engine-host-allowlist="*" \
      --engine-rpc-port=8551      \
      --Xmerge-support=true       \
      --engine-jwt-secret=<path to jwtsecret.hex>
    ```

Specify the path to the `jwtsecret.hex` file generated in [step 2](#2-generate-the-shared-secret) using the
[`--engine-jwt-secret`](../Reference/CLI/CLI-Syntax.md#engine-jwt-secret) option.

See the [`--engine-*`](../Reference/CLI/CLI-Syntax.md#engine-host-allowlist) options for more information on running
Besu as an execution client.

## 4. Generate validator keys and stake ETH

If you're running a [validator client](#beacon-node-and-validator-client), create a test Ethereum address (you can do
this in
[MetaMask](https://metamask.zendesk.com/hc/en-us/articles/360015289452-How-to-create-an-additional-account-in-your-wallet)).
Fund this address with testnet ETH using a faucet.

> You can find a list of [Goerli faucets](https://github.com/eth-clients/goerli#meta-data-g%C3%B6rli) and [Sepolia faucets](https://github.com/eth-clients/sepolia#meta-data-sepolia).

Reviewer: ```suggestion
See the list of [Goerli faucets](https://github.com/eth-clients/goerli#meta-data-g%C3%B6rli) and [Sepolia faucets](https://github.com/eth-clients/sepolia#meta-data-sepolia).
```

To run Teku as a beacon node and validator in a single process, run the following command:

=== "Goerli"

    ```bash
    teku \
      --data-path "datadir-teku"                                \
      --network goerli                                          \
      --ee-endpoint http://localhost:8551                       \
      --ee-jwt-secret-file <path to jwtsecret.hex>              \
      --log-destination console                                 \
      --rest-api-enabled=true                                   \
      --validators-proposer-default-fee-recipient=<ETH address> \
      --validator-keys=<path to key file>:<path to mnemonic file>[,<path to key file>:<path to mnemonic file>,...] \
      --p2p-discovery-bootnodes "enr:-Iq4QMCTfIMXnow27baRUb35Q8iiFHSIDBJh6hQM5Axohhf4b6Kr_cOCu0htQ5WvVqKvFgY28893DHAg8gnBAXsAVqmGAX53x8JggmlkgnY0gmlwhLKAlv6Jc2VjcDI1NmsxoQK6S-Cii_KmfFdUJL2TANL3ksaKUnNXvTCv1tLwXs0QgIN1ZHCCIyk"
    ```

=== "Sepolia"

>     Sepolia is a permissioned network and can't be run using a validator client without [becoming a validator first](https://notes.ethereum.org/zvkfSmYnT0-uxwwEegbCqg).

Reviewer: ```suggestion
    Sepolia is a permissioned network and can't be run using a validator client without [requesting to become a validator first](https://notes.ethereum.org/zvkfSmYnT0-uxwwEegbCqg).
```

        2022-03-21 20:44:12.353 INFO  - Syncing     *** Target slot: 76096, Head slot: 3519, Remaining slots: 72577, Connected peers: 9
        ```

> You can check your validator status by searching your Ethereum address on the [Prater Beacon Chain explorer](https://prater.beaconcha.in/).

Reviewer: Can we add a note that explains that **Prater** and **Goerli** and two names for the same chain?

Author: Goerli's beacon chain is now up and running, so I just removed Prater completely
